pkg/nat/nat.go

```diff
@@ -59,6 +62,21 @@ func ParsePort(rawPort string) (int, error) {
 	return int(port), nil
 }
 
+// ParsePortRange parses the port range string and returns start/end ints
+func ParsePortRange(rawPort string) (int, int, error) {
+	if len(rawPort) == 0 {
+		return 0, 0, nil
+	}
+	start, end, err := parsers.ParsePortRange(rawPort)
+	if err != nil {
+		return 0, 0, err
+	}
+	if end < start {
```

Comment:
Looks like `parser.ParsePortRange()` already checks for `end < start`.
Even if you want to override the error message with a more specific one, this code would not be executed.